corehq/apps/export/templates/export/partials/table.html

```diff
@@ -375,7 +375,7 @@ <h4 data-bind="text: case_type"></h4>
         </div>
         <div data-bind="ifnot: $parent.showOwnership">
           <div data-bind="if: isLocationSafeForUser">
-            <a data-bind="attr: {href: editUrl}" class="btn btn-default">
+            <a data-bind="attr: {href: editUrl}" class="btn btn-default"{% if odata_feeds_over_limit %} data-toggle="modal"{% endif %}>
```

Comment:
This button will be updated to "Clone" for OData, so that's why we're updating it.